### PR TITLE
Update integration sample - GA experiments

### LINKED ISSUE #145: Update integration sample - GA experiments
### Brief bug description

After the [rebranding from Kentico Cloud to Kentico Kontent](https://kontent.ai/blog/moving-to-caas-with-kentico-kontent) there are still some samples that use old naming in branches named `samples/*`.

Until the end of October endpoint on domain `kenticocloud.com` stop working and there will be redirect performed:
* https://docs.kontent.ai/reference/api-changelog#a-api-urls-changed

#### Transfer changes made in https://github.com/Kentico/kontent-sample-app-react/pull/140 to the [**`samples/ga-experiments`**](https://github.com/Kentico/kontent-sample-app-react/tree/samples/ga-experiments) branch

* Update dependency from `kentico-cloud-delivery` to `@kentico/kontent-delivery`
    * adjust the breaking changes
* Update dependency `@chevtek/react-spinners` to `@simply007org/react-spinners`
   * no breaking changes
* Rename Kentico Cloud to Kentico Kontent
* adjust `travis.yml` to use the correct domain for deployment

### PR DESCRIPTION
**Pull request by obulaworld**
### Motivation

Fixes #145 

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

**Comment by obulaworld** [comment]
@Simply007 Is this going to be reviewed soon?

**Comment by Simply007** [comment]
@obulaworld - It is in the queue :-) The Hacktoberfest is killing us with all of the pull requests amount :-)

**Comment by makma** [comment]
I'm on it!

File: src/Pages/Home.js [hunk 1]
```diff
@@ -67,10 +67,14 @@ class Home extends Component {
           twitterImage={home.metadataTwitterImage}
         />
         {home.heroUnit &&
-          home.heroUnit.length && <Banner heroUnit={home.heroUnit[0]} />}
+          home.heroUnit.length &&
```
Comment (makma) [inline src/Pages/Home.js]: Is this necessary?

File: src/Pages/Home.js [hunk 2]
```diff
@@ -79,7 +83,8 @@ class Home extends Component {
           text={this.props.t('moreArticles')}
         />
         {home.ourStory &&
-          home.ourStory.length && <OurStory fact={home.ourStory[0]} />}
+          home.ourStory.value &&
+          home.ourStory.length && <OurStory fact={home.ourStory.value[0]} />}
```
Comment (makma) [inline src/Pages/Home.js]: Is length check/unwrap necessary here?

File: src/Pages/Home.js (hunk 1 above)
Comment (obulaworld) [reply src/Pages/Home.js]: Hi @makma, they are actually not, I have removed it.

File: src/Pages/Home.js (hunk 2 above)
Comment (obulaworld) [reply src/Pages/Home.js]: Hi @makma, It is actually not, I have removed it.

**Pull request merged**